src/waypoint.jsx

```diff
@@ -10,7 +10,6 @@ import {
   INVISIBLE, INSIDE, BELOW, ABOVE,
 } from './constants';
 import debugLog from './debugLog';
-import ensureChildrenIsValid from './ensureChildrenIsValid';
```

Comment:
If this file is unused, can we please also remove it in this PR?

Reply:
Ups, that slipped through the cracks. Fixed now.